site/update.md

```diff
@@ -23,6 +23,7 @@ When running the tool in WLST online mode, the update operation may require serv
 - `101` - The domain needs to be restarted and the Update Domain Tool needs to be re-invoked with the same arguments.
 - `102` - The servers impacted by the update operation need to be restarted, in a rolling fashion, starting with the Administration Server, if applicable.
```

Comment:
the rollback command arg is not documented. And the 101, 102 are not valid are they?

Reply:
The rollback command arg is for future operator use only originally, let me add the doc in it. I didn't change 101 and 102, no idea where they come from